src/HW1.java

```diff
@@ -0,0 +1,12 @@
+class HW1 {
+    int year = 2023;
+
+    public static int MYMETHOD(int a, int b){
```

Comment:
Method should be named informative and describe what it does.
lowerCamelCase should be used for methods naming - sumOfTwoNumbers